### PR TITLE
Yet another try to enable HPA e2e tests for 1.1 release

### PR DESCRIPTION
**Pull request by piosz**
ref #17596
follow up #17639

cc @ixdy @jlowdermilk @ihmccreery

### CODE REVIEW

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -118,14 +118,13 @@ CURRENT_RELEASE_PUBLISHED_VERSION="ci/latest-1.1"
...
 REBOOT_SKIP_TESTS=(
-    "\[Skipped\]"
     "Restart\sshould\srestart\sall\snodes"
     "\[Example\]"
     )
 
 # Specialized tests which should be skipped by default for projects.
 GCE_DEFAULT_SKIP_TESTS=(
-    "${REBOOT_SKIP_TESTS[@]}"
+    "\[Skipped\]"
```
Comment (wojtek-t) [inline hack/jenkins/e2e.sh]: This isn't correct - we definitely want to skip e.g. [Example] in GCE runs.

Comment (piosz) [inline hack/jenkins/e2e.sh]: Fixed. Thanks for catch!

**Comment by wojtek-t** [comment]
LGTM

**Comment by piosz** [comment]
@ixdy @jlowdermilk 
I'd like to enable HPA tests for release 1.1 in disruptive suite. I tried with this PR, #17639 and #17365. Should I cherry-pick those changes to the release branch?

**Comment by jlowdermilk** [comment]
@piosz, yes, although you only need to cherry pick #17639, specifically the changes to [test/e2e/horizontal_pod_autoscaling.go](https://github.com/kubernetes/kubernetes/pull/17639/files#diff-4e2ca00b158a4e196a6549415321de2bL53). Jenkins jobs pull `hack/jenkins/e2e.sh` from head, but anything under `/test/e2e` is built and run from the checked out branch.

**Comment by ixdy** [comment]
I don't understand how, but somehow when this merged, "GCE L7 LoadBalancer Controller should create GCE L7 loadbalancers and verify Ingress" stopped running on kubernetes-e2e-gce-disruptive-1.1.

**Comment by ixdy** [comment]
Aha, figured it out:
```
++ '\[Autoscaling\]\sReplicationController' 'GCE\sL7\sLoadBalancer\sController'
bash: line 658: \[Autoscaling\]\sReplicationController: command not found
```

The comment on [line 653](https://github.com/kubernetes/kubernetes/pull/17718/files#diff-d228e0b744115de6cc5a03d8d8f5fbfaR653) is breaking this.

**Comment by piosz** [comment]
Apologies for that. I'll create a fix.

**Pull request merged**